tests/integration/config_test.go

```diff
@@ -209,5 +209,16 @@ var _ = Describe("odo config test", func() {
 			Expect(configValue).To(Not(ContainSubstring(("PORT"))))
 			Expect(configValue).To(Not(ContainSubstring(("SECRET_KEY"))))
 		})
+
+		It("should list config without logging into the OpenShift cluster", func() {
+			helper.CmdShouldPass("odo", "create", "nodejs", "--project", project)
+			helper.CmdShouldPass("odo", "config", "set", "--env", "hello=world")
+			kubeconfigOld := os.Getenv("KUBECONFIG")
```

Comment:
Does it help on travis CI ?, i guess no because the scenario is not actually being tested on a down cluster in travis. Correct me if i am wrong cc_ @kadel @dharmit 

In travis CI ```os.Getenv("KUBECONFIG")``` will return an empty string because no such ```KUBECONFIG``` is set before or after setting up ```oc cluster up```. Then how it is used for travis CI

If this is true  then can we rewrite it in this way 
```
helper.CmdShouldPass("odo", "create", "nodejs", "--project", project)
helper.CmdShouldPass("odo", "config", "set", "--env", "hello=world")
val, ok := os.LookupEnv("CI")
if ok && val == "openshift" {
	kubeconfigOld := os.Getenv("KUBECONFIG")
       os.Setenv("KUBECONFIG", "")
       configValue := helper.CmdShouldPass("odo", "config", "view")
       os.Setenv("KUBECONFIG", kubeconfigOld)
} else {
       helper.CmdShouldPass("odo", "logout")
       configValue := helper.CmdShouldPass("odo", "config", "view")
       helper.CmdShouldPass("odo", "login")
}
Expect(configValue).To(ContainSubstring("hello"))
Expect(configValue).To(ContainSubstring("world"))
```

Reply:
> Does it help on travis CI ?, i guess no because the scenario is not actually being tested on a down cluster in travis. Correct me if i am wrong cc_ @kadel @dharmit
> 
> In travis CI `os.Getenv("KUBECONFIG")` will return an empty string because no such `KUBECONFIG` is set before or after setting up `oc cluster up`. Then how it is used for travis CI

That is OK, why it would be a problem if it returns an empty string?

Reply:
> The issue that I see here is on the next line - os.Setenv("KUBECONFIG", ""), @dharmit you need to set it to some path (preferably not existing), not to an empty string. Empty string means that the default path will be used usually .kube/config

I was not sure about the use of empty ```KUBECONFIG```. @kadel Thanks for the clarification.

Reply:
Same here. Was a TIL for me. :smile: